CoreFoundation/Base.subproj/CFRuntime.h

```diff
@@ -194,19 +194,11 @@ typedef struct __CFRuntimeBase {
     // This matches the isa and retain count storage in Swift
     uintptr_t _cfisa;
     uintptr_t _swift_rc;
-    // This is for CF's use, and must match _NSCFType layout
-#if defined(__LP64__) || defined(__LLP64__)
+    // This is for CF's use, and must match __NSCFType/_CFInfo layout
```

Comment:
Just one comment: do we want to move `_CFInfo` to match (one `UInt64` instead of two `UInt32`s?) @mundaym

Reply:
By which I mean: in this same patch.

Reply:
I generally try to keep reverts away from other changes, personally. But that comes from a non-git background. In general, this PR, #1694 and #1709 are already getting pretty entangled as it is. 

Is the purpose of this PR to unblock ARM32 caused by a regression, or to generally attempt to fix the underlying issues that caused the regression. If the former, I'd say keep this focused, and then it should be easy to follow up with the other fixes. If the latter, than we should probably have these commits coming in as a group, but then it gets a bit more difficult since I can't really test PPC BE on behalf of @mundaym.

Reply:
Then I’m good with this and let’s figure out the rest step by step.